p3/management/commands/speaker_list.py

```diff
@@ -87,21 +87,21 @@ def handle(self, *args, **options):
         speaker_list.sort(key=speaker_list_key)
 
         # Print list of speakers
-        print ('<h2>Speakers</h2>')
+        self.stdout.write('<h2>Speakers</h2>')
```

Comment:
Wouldn't it be better to go with logger everywhere, and just patch the handler in tests?

Reply:
Well, I've used the `self.stdout` is what the documentation recommends (see the first Note in https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/). And I think it actually makes sense - we want the output to go to stdout not the django instance logger when running the commands.

Reply:
If it's recommended in django docs then it makes sense to follow the practice; In this case it actually probably makes sense to push all the "lines" to some container and then print the the output in one place, joining with "\n". Actually... it would make sense to simply use a template engine here... :)

I probably left this comment under a wrong line though, because this one clearly prints stuff to be later captured and used in some other way, rather than actually mimic logging (as some other lines in this PR do); and in case there are log-like prints, I'm a big fan of actually using logs instead of prints. ;) (like for example it's done in the deployment script)

Reply:
I think you are misreading the purpose of the print()s: they are intended to produce output on stdout to pipe elsewhere. By changing these to stdout.write() you remove the line ends, which are intended to be there for better readability. Your changes remove them.

Reply:
I think there's been some confusion here - this is not using `sys.stdout.write` directly, which would require adding the newlines at the end, as you suggest; It's using `BaseCommand.stdout`, which does append the newline characters by default (see https://github.com/django/django/blob/stable/1.11.x/django/core/management/base.py#L201).